src/Vendr.PaymentProviders.Stripe/StripePaymentProviderBase.cs

```diff
@@ -224,9 +224,7 @@ protected static void ConfigureStripe(string apiKey)
 
         protected string GetTransactionId(PaymentIntent paymentIntent)
         {
-            return (paymentIntent.Charges?.Data?.Count ?? 0) > 0
-                ? GetTransactionId(paymentIntent.Charges.Data[0])
-                : null;
+            return GetTransactionId(paymentIntent.LatestCharge);
```

Comment:
Could latest charge be null here? If so, it should probably check and return a null transaction ID

Reply:
@mattbrailsford Yes it could. But it's null propagated on line 237 already 🙂

Reply:
Fair point 👍